openreview/conference/templates/areachairWebfield.js

```diff
@@ -228,16 +228,21 @@ var formatData = function(blindedNotes, officialReviews, metaReviews, noteToRevi
   });
 };
 
+var displayError = function(message) {
+  message = message || 'The group data could not be loaded.';
+  $('#notes').empty().append('<div class="alert alert-danger"><strong>Error:</strong> ' + message + '</div>');
+};
+
 var getUserProfiles = function(userIds) {
   var ids = _.filter(userIds, function(id) { return _.startsWith(id, '~');});
   var emails = _.filter(userIds, function(id) { return id.match(/.+@.+/);});
 
   var profileSearch = [];
   if (ids.length) {
-    profileSearch.push($.post('/profiles/search', JSON.stringify({ids: ids})));
+    profileSearch.push(controller.post('/profiles/search', {ids: ids}),);
```

Comment:
It would be better to use `Webfield.post`

Reply:
Fixed.